gen3-cli/docker-python-install.sh

```diff
@@ -0,0 +1,3 @@
+pyenv global 3.8.0
```

Comment:
why do we need pyenv in the docker?

Reply:
The codeontap image uses pyenv to install different python versions based on what someone wants to use, saves having to manage all of the python version installs that people want but aren't available via debian package managers etc.